pilot/pkg/kube/inject/app_probe_test.go

```diff
@@ -37,6 +37,7 @@ func TestRewriteAppHTTPProbe(t *testing.T) {
 		{
 			name: "one-container",
 			sidecar: &SidecarInjectionSpec{
+				RewriteAppHTTPProbe: true,
```

Comment:
This doesn't make any sense - why would this be part of the spec, next to containers ?